app/services/user_access_key.rb

```diff
@@ -57,6 +57,7 @@ def xor(ciphertext)
   end
 
   def store_encrypted_key(encrypted_key)
+    @made = true
```

Comment:
I feel like there are a few places in this codebase where we are passing around and assigning ivars like this. 

Is there another way to do this without them? 

In general, I tend to privately `attr_reader` or `attr_accessor` all ivars in classes. I am not sure I have a good reason for this, but this thread has some interesting discussion on the topic: https://forum.upcase.com/t/using-instance-variables-vs-attribute-accessors/1788/2